marlowe-playground-client/src/MainFrame/View.purs

```diff
@@ -81,18 +82,33 @@ render settings state =
               ]
           ]
       , modal state
+      , div [ classes [ ClassName "footer" ] ]
+          [ div [ classes [ flex, ClassName "links" ] ]
+              [ a [ href "https://cardano.org/" ] [ text "cardano.org" ]
+              , vl
+              , a [ href "https://iohk.io/" ] [ text "iohk.io" ]
+              ]
+          , div [] [ text "\x00A9 2020 IOHK Ltd" ]
```

Comment:
For clarity, I'd pull this out to `copyright = text "\x00A9"` (assuming I've guess what that code means correctly). 

(And if I haven't, that's my point. :-D )